gravitee-rest-api-management/gravitee-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApplicationSubscriptionsResource.java

```diff
@@ -110,18 +115,18 @@ public Response createSubscriptionWithApplication(
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = "List subscriptions for the application",
-            notes = "User must have the READ_SUBSCRIPTION permission to use this service")
-    @ApiResponses({
-            @ApiResponse(code = 200, message = "Paged result of application's subscriptions", response = PagedResult.class),
-            @ApiResponse(code = 500, message = "Internal server error")})
+    @Operation(summary = "List subscriptions for the application", description = "User must have the READ_SUBSCRIPTION permission to use this service")
+    @ApiResponse(responseCode = "200", description = "Paged result of application's subscriptions",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = PagedResult.class)))
```

Comment:
Could you extend `PagedResult<SubscriptionEntity>` as you have done in other classes ?